bindgen/clang.rs

```diff
@@ -1439,10 +1442,11 @@ impl Type {
     /// Given that this type is an array or vector type, return its number of
     /// elements.
     pub(crate) fn num_elements(&self) -> Option<usize> {
-        let num_elements_returned = unsafe { clang_getNumElements(self.x) };
-        if num_elements_returned != -1 {
-            Some(num_elements_returned as usize)
+        let n = unsafe { clang_getNumElements(self.x) };
+        if n >= 0 {
+            Some(n.try_into().expect("array size does not fit in `usize`"))
         } else {
+            debug_assert_eq!(n, -1);
             None
```

Comment:
Same pattern

Reply:
Here it's not quite the same since this returns `usize`, so there's two different checks: `>= 0` and `c_longlong` -> `usize`, where for `c_int` to `c_uint` above these basically are the same check.